framework/src/org/apache/cordova/PluginManager.java

```diff
@@ -577,4 +580,19 @@ public Bundle onSaveInstanceState() {
         }
         return state;
     }
+
+    /**
+     * Collect all plugins PathHandlers
+     * @return list of PathHandlers in no particular order
+     *
```

Comment:
```suggestion
     * Collect all plugins PathHandlers
     *
     * @return list of PathHandlers in no particular order
```

Move new line above or delete new line.